pontoon/base/static/js/main.js

```diff
@@ -609,7 +605,7 @@ var Pontoon = (function (my) {
       }
 
       // Machine translation (Caighdean)
-      if (self.locale.code === 'ga-IE') {
+      if (!customSearch && self.locale.code === 'ga-IE') {
```

Comment:
On custom search (also used on `/machinery`) we don't have an entity.id, so we cannot use this service.